### PR TITLE
added docs section to readme

### PR DESCRIPTION
**Pull request by mhowlett**
beefed out the wiki. adding a reference to it here, and other docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ confluent-kafka-dotnet is derived from Andreas Heider's [rdkafka-dotnet](https:/
 We're fans of his work and were very happy to have been able to leverage rdkafka-dotnet as the basis of this
 client. Thanks Andreas!
 
+## Documentation
+
+* [Github Wiki](https://github.com/confluentinc/confluent-kafka-dotnet/wiki)
```
Comment (edenhill) [inline README.md]: drop "Github", not relevant and slightly confusing.

File: README.md [hunk 2]
```diff
@@ -31,6 +31,14 @@ confluent-kafka-dotnet is derived from Andreas Heider's [rdkafka-dotnet](https:/
...
+* [Blog Posts and Presentations](https://github.com/confluentinc/confluent-kafka-dotnet/wiki/Presentations-&-Blogs)
+* [Hosted by Confluent](https://docs.confluent.io/current/clients/dotnet.html)
```
Comment (edenhill) [inline README.md]: Unclear. How about "Official (and usually outdated) .NET client documentation"

Comment (mhowlett) [reply README.md]: i love the casual nature of that suggestion, sold.

**Comment by mhowlett** [comment]
ironic that a (successful!) effort aimed at demonstrating that 'good process' can sometimes be counter productive, is blocked from being discoverable by good process.

**Pull request closed without merging**